ui/src/app/pages/signed-in/component.ts

```diff
@@ -88,17 +89,20 @@ export class SignedInComponent implements OnInit, OnDestroy, AfterViewInit {
       });
     });
 
-    this.signInService.isSignedIn$.subscribe(signedIn => {
+    this.signOutNavigateSub = this.signInService.isSignedIn$.subscribe(signedIn => {
       if (!signedIn) {
         this.signOut();
       }
     });
+    this.subscriptions.push(this.signOutNavigateSub);
```

Comment:
What does the `subscriptions` array do in this component?

Reply:
It's purely for component teardown purposes: https://github.com/all-of-us/workbench/blob/56a6c34fb66952234d4607d15773c404ca44e8f9/ui/src/app/pages/signed-in/component.ts#L215-L217